### ISSUE TITLE
Possible bug with version 9.0

### ISSUE THREAD

**Issue by MikeyJN** [FrDH/mmenu-js#1261]
I could be doing something wrong here, but I've having problems getting the very latest version to work correctly.

I've set-up afiddle using v9.0.1 and the example code from the 'mmenu' website and as you can see, the menu will open OK, but upon closing there is a shift in layout and then the menu will not open again: https://jsfiddle.net/h82mLd3y/

Here is an another fiddle using **identical** code, but with v8.5.24 and it's working perfectly: https://jsfiddle.net/5ast1z4p/

Is this a possible bug, or have I gone wrong somewhere?

Many thanks,

Mike

**Comment by andreanironi** [comment]
I have the very same issue, and it works when pressing the ESC button rather than clicking on the right area. I'll try to submit a patch later if this is the case, unless it's elated to a non-backward compatible behavior of the 9.x branch

**Comment by MikeyJN** [comment]
![img1](https://user-images.githubusercontent.com/64698829/144886616-9b127bfa-2dc5-4f95-b668-058d8ebfe430.png)

Another issue I've found on the mmenu.js website which I'm not sure is also related to v9.0, but when viewed on a large'ish/wide monitor, the page loads with the menu already opened which doesn't then close when the 'X' is clicked on. Also there is overflow visible hence the horizontal scrollbar - see IMAGE attached...

**Comment by MikeyJN** [comment]
Hi FrDH,

Have you got a time estimate as to when the issues with v9.0 might be resolved?!


Many thanks,

Mike

**Comment by FrDH** [comment]
I did not have the change to find out why, but if you add an ID to the `DIV.page`, it seems to work.
````
<div id="page" class="page">
  <div id="header">
    <a href="#menu">OPEN</a>
  </div> 
</div>
````

**Comment by FrDH** [comment]
Should be fixed in 9.0.3
